packages/phrasebook/__test__/locale.test.js

```diff
@@ -64,7 +64,7 @@ const assembleImports = () => {
       let componentsList = getDirList(`./${DEFAULT_LANG_DIR}/${lang}`);
       if(componentsList) {
         for (const component of componentsList) {
-          let imported = require(`../${DEFAULT_LANG_DIR}/${lang}/${component}`);
+          let imported = import(`../${DEFAULT_LANG_DIR}/${lang}/${component}`);
```

Comment:
dynamic import returns `Promise`. Shall we `await` here?

Reply:
I'm not sure if this is being resolved on `done()` after this function is called or not but this is how `v7` is currently. Let me do a bit more investigation on this.

Reply:
indeed, `v7` uses `await`. I still think this is the way. We're better off without `done()` 😄

Reply:
Updated. I'll create another PR for `v7` to use `async/await` as well.